### DISCUSSION TITLE
How lammps use the potential function trained by dpgen

### DISCUSSION THREAD

**Discussion by manbujin** [deepmodeling/dpgen#986] [Q&A]
Hello,
  I've trained four potential （graph.000.pb  graph.001.pb graph.002.pb graph.003.pb）functions through dpgen,  and I'd like to ask if I want to use them all, or choose one of them.
  In the “model_devi.out” file it works normally, so I would like to ask if it is necessary to compare these four potential functions.
And please ask about the specific steps of dpgen's trained potential function for lammps, there is no relevant example found on the Internet.
Thank you so much！

**Comment by wanghan-iapcm** [comment]
the models trained by dpgen are typically trained with a relatively small number of training steps. The accuracy of force should be fine but that of the energy may not be satisfactory. 

For productive purpose, you may want to train a model with the data generated by dpgen for a large number of training steps.

**Reply by manbujin** [reply]
Yes, I want to do a classical molecular dynamics simulation with the potential function generated by dpgen.
I also have a question I would like to ask, if I train a potential function with five elements, such as (TiFeCrWV), can this potential function be used directly in a quaternary system, such as (TiFeCrV).
